css-transitions-2/Overview.bs

```diff
@@ -167,6 +167,124 @@ Each time a new transition is generated, the current value of the (already
+could not be started on an element that was not being rendered for the previous
+[=style change event=] (see: [[css-transitions-1#starting]]).
+
+In some cases it makes sense to start transitions on newly inserted elements or
+elements that change from not [=being rendered=] to being rendered. To allow
+for that, this specification introduces ''@starting-style''.
+
+The <dfn at-rule id="at-ruledef-starting-style">@starting-style</dfn> rule is a
+grouping rule where the rules inside it is used to establish styles to
+transition from if the previous [=style change event=] did not establish a
+[=before-change style=] for the element whose styles are being computed.
+
+Note: This means that ''@starting-style'' rules only apply to some elements
+during a computed style update, namely elements that were not rendered or part
+of the DOM during the previous [=style change event=].
+
+Define <dfn>starting style</dfn> for an element as the [=after-change style=]
+with ''@starting-style'' rules applied in addition. If an element does not have
+a [=before-change style=] for a given [=style change event=], the
+[=starting style=] is used instead of the [=before-change style=] to compare
+with the [=after-change style=] to start transitions
+([[css-transitions-1#starting]]).
+
+The rules inside ''@starting-style'' cascade as any other grouping rules without
+introducing any new ordering to the cascade, which means rules inside
+''@starting-style'' do not necessarily win over those outside.
+
+Style rules in ''@starting-style'' do not apply to [=after-change style=].
+Thus, the presence of matching rules in ''@starting-style'' can cause
+transitions to occur on elements that otherwise could not have transitions
+because they lack a [=before-change style=].
+
+''starting style'' inherits from the parent's [=after-change style=] just like
+[=after-change style=] does.
+
+<div class=example>
+
+  Transitioning background-color of an h1 element from transparent to green
+	from initial rendering:
+
+	<pre class=lang-css>
+	h1 {
+		transition: background-color 1.5s;
+		background-color: green;
+	}
+	@starting-style {
+		h1 {
+			background-color: transparent;
+		}
+	}
+	</pre>
+
+	Conditional rules can be used with CSS Nesting:
+
+	<pre class=lang-css>
+	h1 {
+		transition: background-color 1.5s;
+		background-color: green;
+		@starting-style {
+			background-color: transparent;
+		}
+	}
+	</pre>
+</div>
+
+<div class=example>
+
+	Transitioning an element with opacity in and out of display:none:
+
+	<pre class=lang-css>
+	#target {
+		transition-property: opacity, display;
+		transition-duration: 0.5s;
+		display: block;
+		opacity: 1;
+		@starting-style {
+			opacity: 0;
+		}
+	}
+	#target.hidden {
+		display: none;
+		opacity: 0;
+	}
+	</pre>
+
+	The display is transitioning to allow for an opacity transition before
+	flipping from <code>display:block</code> to <code>display:none</code>.
+
+	Setting ''@starting-style'' opacity to 0 means the element will
```

Comment:
This seems a little too short, and I'd also prefer to avoid the word "set", so I'd say: `Specifying ''opacity: 0'' in the ''@starting-style'' rule means`...

(Also note additional quoting of CSS properties and values.)